packages/core/src/FormField.js

```diff
@@ -47,7 +49,8 @@ const FormField = props => {
         transitionDuration: '.1s',
         opacity: showLabel ? 1 : 0,
         pointerEvents: 'none',
-        position: 'relative'
+        position: 'relative',
+        width: `calc(100% - ${ml})`
```

Comment:
(n-b) Any perf issues from this calc? We render many form fields on some pages.

Reply:
I looked into this concern a bit. This will not have a noticeable impact on performance, even when used in bulk.